packages/dmg-builder/package.json

```diff
@@ -13,9 +13,9 @@
 		"vendor"
 	],
 	"dependencies": {
-		"app-builder-lib": "22.11.7",
-		"builder-util": "22.11.7",
-		"builder-util-runtime": "8.7.6",
+		"app-builder-lib": "22.11.8",
+		"builder-util": "22.11.8",
```

Comment:
Should these also be changed to `"workspace:*"` just like `builder-util-runtime` is doing so?

Reply:
Nice catch. Indeed.